web/gds-user-ui/src/modules/dashboard/certificate/lib/trixoQuestionnaireValidationSchema.ts

```diff
@@ -14,16 +14,15 @@ export const trixoQuestionnaireValidationSchema = yup.object().shape({
     has_required_regulatory_program: yup.string().oneOf(['no', 'yes']).default('no'),
     conducts_customer_kyc: yup.boolean().default(false),
     kyc_threshold: yup
-      .number()
+      .string()
       .transform((value, originalValue) => {
         if (originalValue) {
-          console.log('[originalValue]', originalValue);
-          const v = originalValue?.replace(/^0+/, '');
+          const stringValue = typeof originalValue === 'string' ? originalValue : originalValue.toString();
+          const v = stringValue?.replace(/^0+/, '');
           return v?.length > 0 ? Number(v) : 0;
         }
         return value;
-      })
-      .default(0),
```

Comment:
In the payload, I verified that the default is 0 if a user doesn't enter a value and that the type is a number.